apps/design/frontend/src/signature_image_input.tsx

```diff
@@ -0,0 +1,44 @@
+import styled from 'styled-components';
+import { images } from '@votingworks/ui';
+import { ImageInput, ImageInputProps } from './image_input';
+
+const StyledImageInput = styled(ImageInput)`
+  img {
+    width: 10rem;
```

Comment:
This means the precinct split image will now be larger, as it was previously set to 3rem, which was very hard to see. The Seal right above it was 10rem, so now they are consistent.